stable/insights-agent/templates/rbac-reporter/rbac.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.releasewatcher.enabled -}} 
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "insights-agent.fullname" . }}-rbac-reporter
+  labels:
+    app: insights-agent
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: edit
```

Comment:
Where is this `ClusterRole` coming from? I'm noticing now we have the same `ClusterRole` in `release-watcher/rbac.yaml` - where is it defined?

We shouldn't need edit permissions for this or release-watcher, just view permissions.

Reply:
changed for `rbac-reporter`. For the case of release watcher the discussion was [ here](https://github.com/FairwindsOps/charts/pull/182#pullrequestreview-323005723)

Reply:
also looking at this chart rbac https://github.com/FairwindsOps/release-watcher/blob/master/chart/templates/rbac.yaml ... should we replicate from this?